README.md

```diff
@@ -54,6 +54,85 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Group Files
```

Comment:
is this just an apiary thing, or does putting the word "group" in front of everything add extra value to the reader?